scan/lib/scan/test_command_generator.rb

```diff
@@ -95,6 +95,14 @@ def xcodebuild_log_path
         return File.join(containing, file_name)
       end
 
+      # Generate destination parameters
+      def destination
+        unless Scan.cache[:destination]
+          Scan.cache[:destination] = [*Scan.config[:destination]].map { |dst| "-destination '#{dst}'" }.join(' ')
```

Comment:
Not sure what you mean with https://github.com/fastlane/fastlane/pull/4758#discussion_r64121544? Instead of `[]` we should use `Array()`?